umbraco/Umbraco.UI
# Upgrade storybook version

## Description

Upgraded storybook to version 7.1.0.

## Types of changes




## Motivation and context

Storybook now offers support for figma.

## How to test?

Verify that different storybook files load correct and no new console errors are presented.

## Screenshots (if appropriate)

## Checklist
